src/pages/index.js

```diff
@@ -1,6 +1,7 @@
 import { lazy } from 'react';
 
 export const imports = {
+  Dashboard: () => import(/* webpackChunkName: "recipe" */ 'pages/Dashboard'),
```

Comment:
I think this chunk name should be "admin" or "dashboard", rationale being that 99% of users who load the recipe chunk will never use this code - so it should be in its own chunk.